Modules/TOF/include/Base/Counter.h

```diff
@@ -44,9 +44,12 @@ class Counter
   ~Counter() = default;
 
   /// Functions to increment a counter
-  /// @param v Index in the counter array to increment, @param weight weight to add to the array element
-  void Add(const unsigned int& v, const unsigned int& weight);
+  /// @param v Index in the counter array to increment
+  /// @param weight weight to add to the array element
+  void Add(const unsigned int& v, const uint32_t& weight);
```

Comment:
why not change both to uint32_t ? `v` is not really clear as a name, perhaps you could change it.

Reply:
I agree, I also realized it while opening the PR, let me change this

Reply:
For the type I used unsigned int because this is mostly used in for loops, it can be templated of course, @Barthelemy what do you think?

Reply:
@njacazio Yes, that makes sense.